docs/entities/serviceclaim.md

```diff
@@ -0,0 +1,49 @@
+# ServiceClaim
+
+A Service Claim represents a claim for Registered Service.
+
+## Specification
+
+The definition of a ServiceClaim can be obtained directly from our [ServiceClaim
+CRD](../../config/crd/bases/primaza.io_serviceclaims.yaml). The specification
+contains two required properties:
+
+- ServiceClassIdentity: A set of key/value pairs that identify the service
+  class. Examples of service class identity keys include type of service, and
+  provider of service. This property is required.
+- ServiceEndpointDefinitionKeys: An array of keys that is required for
+  connectivity. The values corresponding to each of these keys will be extracted
+  from the service. This property is required.
+
+There are other three optional fields. The EnvironmentTag and
+ApplicationClusterContext are mutually exclusive:
+
+- Application: Fields indentifies application resources through kind, apiVersion and label selector. 
+- EnvironmentTag: A string representing one of the environment.
+- ApplicationClusterContext: A combination of environment name and namespaces.
```

Comment:
Hum... not sure that is what it is. It is specific cluster and namespace. It is mutually exclusive with Environment Tag that points to multiple clusters and namespaces.

Reply:
How about `A combination of ClusterEnvironment resource name and namespace`?